concepts/discovery.md

```diff
@@ -27,8 +27,9 @@ Each discovery result also has a timestamp when it was added to or updated in th
 Discovery results can either be ignored or approved, where in the latter case a Thing is created for them and they become available in the application. 
 If an entry is ignored, it will be hidden in the inbox without creating a Thing for it. 
 
-openHAB offers a service to automatically ignore discovery results in the inbox, whenever a Thing is created manually, that represents the same Thing, as the respective discovery result would create. 
-This Thing would either have the same Thing UID or the value of its representation property is equal to the representation property's value in the discovery result.
+openHAB offers a service to automatically ignore discovery results in the inbox, whenever a) a Thing has been created manually, that represents the same Thing as the respective discovery result would create, or b) whenever a Thing has been discovered separately by two alternate discovery services e.g. by mDNS and UPnP.
```

Comment:
I am not sure, if this is the right place for your contents.
This is clearly a developer topic and the concepts articles are also aiming for users to read.
Also the identical content is written down in this pr within the developer section (where it belongs to in my view)

@cweitkamp can you share your opinion here?

Reply:
@Confectrician, you did not ask, but here are my two cents.  given that this is a high level concepts document, and given that the topic is discovery, I would not throw terms such as mDNS or UPnP in here.

Reply:
Two things..
- You suggest that my proposed change is too complex for users. And I partly agree with you. However the original text DOES say _"representation property is equal to the representation property's value in the discovery result"_ and I think that if you open a can of worms by mentioning representation Property then its description must be correct, precise, and complete. OTOH the alternate solution is to delete the mention of representation property from this page. Either way, IMHO, this page needs an edit.
- In any case, I can remove the the mDNS / UPnP part; no problem.

Reply:
@andrewfg, yep - the concept of a representation property is not exactly something that is common knowledge.  Agree it needs to be completely explained, or the level of the article needs to be taken up a level so that you don't have to use such a special term.

Reply:
I simplified the text. Please check.

Reply:
LGTM

Reply:
I guess this is already clear.